Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -185,6 +185,9 @@ def parse_response(resp, err_operation):
         if res_json.get("message").find('Requested sample not found') != -1:
             demisto.results(err_msg)
             sys.exit(0)
+        elif res_json.get("message").find("AF Cookie Not Found") != -1:
+            demisto.results(err_msg)
```

Comment:
if there are no results, 
please return "No entries found that match the query" as @kirbles19 suggested

Reply:
@yuvalbenshalom it's in a different part in the code, changed.